src/main/java/view/DashboardView.java

```diff
@@ -94,46 +109,29 @@ public void actionPerformed(ActionEvent evt) {
                 new ActionListener() {
                     public void actionPerformed(ActionEvent evt) {
                         if (evt.getSource().equals(reset)) {
-                            viewManagerModel.setActiveView("Reset Balance");
+                            resetBalanceViewModel.firePropertyChanged();
+                            viewManagerModel.setActiveView(resetBalanceViewModel.getViewName());
                             viewManagerModel.firePropertyChanged();
                         }
                     }
                 }
         );
 
-        quit.addActionListener(
-                new ActionListener() {
-                    public void actionPerformed(ActionEvent evt) {
-                        if (evt.getSource().equals(quit)) {
-                            // TODO: Implement quit.
-                            return;
-                        }
-                    }
-                }
-        );
-
         history.addActionListener(
                 new ActionListener() {
                     public void actionPerformed(ActionEvent evt) {
                         if (evt.getSource().equals(history)) {
-                            viewManagerModel.setActiveView("Transaction History");
-                            viewManagerModel.firePropertyChanged();
+                            // TODO: implement once Transaction History has either a use case or a view
+                            return;
                         }
                     }
                 }
         );
 
-        // TODO: Display user stats.
+        // TODO: Display user stats, once they are available.
 
         // TODO: Display table of information about stocks owned by the user.
 
-        // Pay attention to the input from the buttons in this view.
-        buy.addActionListener(this);
-        sell.addActionListener(this);
-        news.addActionListener(this);
-        reset.addActionListener(this);
-        quit.addActionListener(this);
```

Comment:
Not necessary

Reply:
I agree, I thought I had already removed those lines.

Reply:
Wait do you mean it is not necessary to remove the lines?